### ISSUE TITLE
Produce BIDS Derivatives-compatible outputs

### ISSUE THREAD

**Issue by tsalo** [Brainhack-Donostia/ica-aroma-org#20]
We have a set of outputs of interest from AROMA, including:

- Denoised 4D image, in native and standard space (possibly)
- Metrics (feature scores)
- Component time series
- Component maps, in native and standard space (possibly)
- Classifications (included with feature scores in "classification_overview.txt")
- Figure

We should figure out how best to make those outputs BIDS Derivatives-compatible.

At minimum, I think we can use filenames that are basically BIDS-ish, like what I propose in https://github.com/ME-ICA/tedana/pull/574. This means using entities and suffixes that match BIDS convention, minus the "source entities" from the original files (e.g., `sub`, `ses`, `run`).

I don't know if we want to output the classifications/metrics as a json (as in `tedana`) or a tsv. A tsv would be easier to read...

**Comment by tsalo** [comment]
I'm currently leaning toward the following outputs:

- desc-ICA_decomposition.json
- desc-ICA_mixing.tsv
- desc-ICA_components.nii.gz
- desc-ICAGammaThresholded_components.nii.gz
- desc-smoothAROMAnonaggr_bold.nii.gz
- AROMAnoiseICs.csv (if we keep them)
- desc-AROMA_metrics.tsv (instead of the json)
- desc-AROMA_metrics.json (metadata about metrics)
- aroma.svg

**Comment by eurunuela** [comment]
What about the folder structure? My guess is the users will run it in their data folder, say `sub-001/ses-01/func/`. We should create a derivatives folder or something right?

**Comment by tsalo** [comment]
Since we're planning to use this as a node in a BIDS App (e.g., fMRIPrep), I think we should dump things into an arbitrarily-named output folder. We can always develop a BIDS App version of the CLI at some point, if there's a demand. It's in the BIDS App version where I think we'd want to try to extract subject and session information, and then develop a folder and filename structure around that.

**Comment by oesteban** [comment]
> Since we're planning to use this as a node in a BIDS App (e.g., fMRIPrep), I think we should dump things into an arbitrarily-named output folder.

Yes, it is trivial then to have some manager code that transfers everything to the right position and applies bids-derivatives naming.

> I'm currently leaning toward the following outputs:
> 
> * desc-ICA_decomposition.json
> * desc-ICA_mixing.tsv
> * desc-ICA_components.nii.gz
> * desc-ICAGammaThresholded_components.nii.gz
> * desc-smoothAROMAnonaggr_bold.nii.gz
> * AROMAnoiseICs.csv (if we keep them)
> * desc-AROMA_metrics.tsv (instead of the json)
> * desc-AROMA_metrics.json (metadata about metrics)
> * aroma.svg

Yes, don't make it overly complicated - this should do!
